src/core/announcements/serialization.ts

```diff
@@ -17,50 +14,9 @@ export const serialize = (announcement: Announcement): string => {
     .reduce((serialization, [key, value]) => `${serialization}${key}${value}`, "");
 };
 
-/**
- * serializeToHex() takes in just about anything and attempts to coerce the value to DSNP hexadecimal standard
- *
- * @param value - DSNP User Id/URI BigNumber BigInt number and even unknown
- * @throws {@link InvalidHexadecimalSerialization}
- * Thrown if the coercion failed
- * @returns A hexadecimal serialized value
- */
-export const serializeToHex = (
```

Comment:
This is one of the big changes. The hex serialization is no longer needed as we are not trying to serialize a wide range of values and strip leading zeros.

This is mostly replaced not by `serializeValue`, but `convertToDSNPUserId` and `convertToDSNPUserURI` from #158